src/wechaty/user/room.py

```diff
@@ -390,7 +390,7 @@ async def add(self, contact: Contact) -> None:
 
     async def delete(self, contact: Contact) -> None:
         """
-        delete room
+        Delete contact in a room
```

Comment:
this change is tooo small, can you make a more significant contribution for python-wechaty.

Reply:
I'm very busy with my studies now. I'll contribute again in May

Reply:
I believe that there works are easy for you, and you can get more about google code style of python.  Let's keep eyes on this pr, and I'm waiting for your great commits.